hassio/src/ingress-view/hassio-ingress-view.ts

```diff
@@ -102,14 +99,14 @@ class HassioIngressView extends LitElement {
         }
         if (!addonInfo.version) {
           await showAlertDialog(this, {
-            text: this.supervisor.localize("my.error_addon_not_installed"),
+            text: this.hass.localize("supervisor.my.error_addon_not_installed"),
```

Comment:
Didn't we strip the supervisor translations from the frontend translations?

Reply:
Yes, looks like this PR is missing the filechange to change that.
And the supervisor localize is still initialized, it should not be